features.yaml

```diff
@@ -275,7 +276,7 @@ features:
       maturity: Experimental
       nextExpectedPromotion: ""
     area: Security and policy enforcement
-  - name: "In-Cluster Operator"
+  - name: "Istio Operator"
```

Comment:
The old name was intentional to disambiguate the `IstioOperator` API

Reply:
Ah Alright, will revert this to the old name then, thanks